Podfile

```diff
@@ -49,17 +49,29 @@ target 'WordPressAuthenticatorTests' do
   pod 'OCMock', '~> 3.4'
   pod 'Expecta', '1.0.6'
   pod 'Specta', '1.0.7'
+end
+
+abstract_target 'CI' do
+  pod 'SwiftLint', '~> 0.49'
+end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |configuration|
+      # Let Pods targets inherit deployment target from the app
+      # This solution is suggested here: https://github.com/CocoaPods/CocoaPods/issues/4859
+      pod_ios_deployment_target = Gem::Version.new(configuration.build_settings['IPHONEOS_DEPLOYMENT_TARGET'])
+      configuration.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET' if pod_ios_deployment_target <= ios_deployment_target
 
-  post_install do |installer|
-    # Let Pods targets inherit deployment target from the app
-    # This solution is suggested here: https://github.com/CocoaPods/CocoaPods/issues/4859
-    # =====================================
-    #
-    installer.pods_project.targets.each do |target|
-      target.build_configurations.each do |configuration|
-        pod_ios_deployment_target = Gem::Version.new(configuration.build_settings['IPHONEOS_DEPLOYMENT_TARGET'])
-        configuration.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET' if pod_ios_deployment_target <= ios_deployment_target
```

Comment:
The indentation change here is because I extracted the hook to the global context (I guess you could call it that? _Outside of the `target` block where it was before_).